### ISSUE TITLE
Remove `hashedMessage` and `HashedMessage` references

### ISSUE THREAD

**Issue by matgnt** [International-Data-Spaces-Association/ids-specification#204]
Related to #176 
I still found references, but only in non-*.md files
I think those need to be removed because this is not described:

```
$ git grep -l 'ashed' 
negotiation/message/contract-agreement-verification-message.json
negotiation/message/diagram/contract-agreement-verification-message.puml
negotiation/message/schema/contract-agreement-verification-message-schema.json
negotiation/message/shape/contract-agreement-verification-message-shape.ttl
releases/v0.8/negotiation/message/contract-agreement-verification-message.json
releases/v0.8/negotiation/message/schema/contract-agreement-verification-message-schema.json
releases/v0.8/negotiation/message/shape/contract-agreement-verification-message-shape.ttl

```

**Comment by matgnt** [comment]
Meeting result today:

Wait for #184 
and do NOT change content in folder `releases/v0.8/`

**Comment by sebbader-sap** [comment]
Fixed with #214
